docker-images/kafka-base/Dockerfile

```diff
@@ -23,11 +23,11 @@ ENV VERSION ${version}
 
 # Downloading/extracting Apache Kafka
 RUN curl -O https://www.apache.org/dist/kafka/$KAFKA_VERSION/kafka_$SCALA_VERSION-$KAFKA_VERSION.tgz \
```

Comment:
If you're consistently switching to `${FOO}` rather than `$FOO` then you should change this line too.

Reply:
As you can see I'm rather inconsistent with it :-). This should be fixed now. IMHO especially the `$SCALA_VERSION-$KAFKA_VERSION` combination seems to be quite hard to read.